git/git.go

```diff
@@ -49,9 +50,45 @@ func Add(files []string) {
 	}
 }
 
+func getStaged() []string {
+	out, err := exec.Command("git", "diff", "--name-only", "--staged").Output()
+	lib.Check(err)
+	staged := strings.Split(string(out), "\n")
+	var ret []string
+	for i := range staged {
+		curr := strings.TrimSpace(staged[i])
+		if curr != "" {
+			ret = append(ret, curr)
+		}
+	}
+	return ret
+}
+
+func confirmCommit(stdin io.Reader) bool {
+	fmt.Printf("Would you still like to commit? ('y' or 'n')> ")
+	reader := bufio.NewReader(stdin)
+	response, err := reader.ReadByte()
+	lib.Check(err)
+	return (response == 'y')
+}
+
 // Commit performs a git commit on added files
 // * tested
+// ! reminders untested
```

Comment:
I'm guessing you're going to write the tests before merging? If that is the case, I'm going to hold off on approving until your tests are sorted. Let me know when that happens!

Reply:
Oh I forgot about testing this. It's mostly just wrapping the tested remind function but there is a little more involved logic so I'll test it separately. I will re-request you when that's done.